component/src/main/java/io/siddhi/extension/io/grpc/sink/GrpcSink.java

```diff
@@ -322,24 +316,6 @@ public void onError(Throwable t) {
                 public void onCompleted() {
                 }
             };
-            this.channel = managedChannelBuilder.build();
-            rpcMethod = getRpcMethod(serviceConfigs, siddhiAppName, streamID);
-            this.asyncStub = createStub(serviceConfigs);
```

Comment:
You can have createStub() in init level don't move it to connect

Reply:
need to create the channel before creates the stub, the channel is being created at the 'connect' method, so stub also has to be created at the 'connect' method.